src/config/settings_schema.json

```diff
@@ -2,10 +2,10 @@
   {
     "name": "theme_info",
     "theme_name": "Slate",
-    "theme_version": "1.0.0",
+    "theme_version": "0.10.0",
     "theme_author": "Shopify",
     "theme_documentation_url": "https://github.com/Shopify/slate",
-    "theme_support_email": "theme-support@shopify.com"
+    "theme_support_url": "https://shopify.github.io/slate/"
```

Comment:
Should this just be `theme_support_email: slate@shopify.com` instead? The `theme_documentation_url` is already there above.